sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/SymptomsTabSteps.java

```diff
@@ -0,0 +1,273 @@
+        .feelingIll(webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(FEELING_ILL_OPTIONS))
+        .musclePain(webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(MUSCLE_PAIN_OPTIONS))
+        .fever(webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(FEVER_OPTIONS))
+        .shivering(webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(SHIVERING_OPTIONS))
+        .acuteRespiratoryDistressSyndrome(
+            webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(
+                ACUTE_RESPIRATORY_DISTRESS_SYNDROME_OPTIONS))
+        .oxygenSaturationLower94(
+            webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(
+                OXYGEN_SATURATION_LOWER_94_OPTIONS))
+        .cough(webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(COUGH_OPTIONS))
+        .pneumoniaClinicalOrRadiologic(
+            webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(
+                PNEUMONIA_CLINICAL_OR_RADIOLOGIC_OPTIONS))
+        .difficultyBreathing(
+            webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(
+                DIFFICULTY_BREATHING_OPTIONS))
+        .rapidBreathing(
+            webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(RAPID_BREATHING_OPTIONS))
+        .respiratoryDiseaseVentilation(
+            webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(
+                RESPIRATORY_DISEASE_VENTILATION_OPTIONS))
+        .runnyNose(webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(RUNNY_NOSE_OPTIONS))
+        .soreThroat(webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(SORE_THROAT_OPTIONS))
+        .fastHeartRate(
+            webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(FAST_HEART_RATE_OPTIONS))
+        .diarrhea(webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(DIARRHEA_OPTIONS))
+        .nausea(webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(NAUSEA_OPTIONS))
+        .lossOfSmell(
+            webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(LOSS_OF_SMELL_OPTIONS))
+        .lossOfTaste(
+            webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(LOSS_OF_TASTE_OPTIONS))
+        .otherNonHemorrhagicSymptoms(
+            webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(
+                OTHER_NON_HEMORRHAGIC_SYMPTOMS_OPTIONS))
+        .symptomsComments(
+            webDriverHelpers.getValueFromWebElement(OTHER_NON_HEMORRHAGIC_SYMPTOMS_INPUT))
+        .firstSymptom(webDriverHelpers.getValueFromCombobox(FIRST_SYMPTOM_COMBOBOX))
+        .dateOfSymptom(getDateOfSymptomOnset())
+        .build();
+  }
+
+  private LocalDate getDateOfSymptomOnset() {
+    String dateOfReport = webDriverHelpers.getValueFromWebElement(DATE_OF_SYMPTOM_INPUT);
+    return LocalDate.parse(dateOfReport, DATE_FORMATTER);
+  }
+
+  public void selectMaximumBodyTemperatureInCCombobox(String maximumBodyTemperatureInCCombobox) {
```

Comment:
should we make all methods related to this class private?

Reply:
yes it can be done, we will take this info after the 2 months

Reply:
ok